src/pages/Poster.tsx

```diff
@@ -46,16 +51,22 @@ const Poster = () => {
   const countDownRef = useRef<any>(null)
   const history = useHistory()
 
-  useStorage()
-
-  const [isPosterExpired, setIsPosterExpired] = useState(false)
-  const [secondsRemaining, setSecondsRemaining] = useState(QUESTION_TIME)
-
   const { loading, error, data, refetch } = useQuery<RandomMovieData>(
     RANDOM_MOVIE,
   )
 
-  const { setTimeRemaining, setCurrentImdbId } = useStore()
+  const { timeRemaining, setTimeRemaining, setCurrentImdbId } = useStore()
```

Comment:
I moved timeRemaining into the global store such that we don't need to do awkward updating in hook or transition callbacks.